packages/preset-uno/src/rules/index.ts

```diff
@@ -108,6 +108,7 @@ export const rules: Rule[] = [
   filters,
   tables,
   transforms,
+  screenReadersAccess,
```

Comment:
I guess this should go in the beginning of the list so other rules can override it.

Reply:
I moved it to the beginning, please check! Thanks for your quick reply, highly appreciated!